### PR TITLE
Enable k8s-escape-room to work within Github codespaces

### LINKED ISSUE #50: Enable k8s-escape-room to work within Github codespaces
In order to ease the access to the k8s-escape-room, we want to make it usable within Github codespaces.

### PR DESCRIPTION
**Pull request by steadyk**
This PR adapts the k8s-escape-room to work in a Github codespaces environment.

**Changes:**
* add documentation for usage within Github codespaces
* move local and Github codespaces usage documentation to own files
* add slightly adapted workbench for codespaces
* update terminal image (remove all personal data)

**To test:**
* documentation is understandable
* works in Github codespaces (already tested)

Fixes #50

### CODE REVIEW

File: .docs/codespaces.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+```
+
+Wait a short time while everything is being set up.
+
+![K8s Escape Room in terminal](../.images/k8s-escape-room-terminal.png)
+
+## Play
+As soon as the cluster and its resources are ready you can start playing.
+
+To access the escape room UI in your Github codespace, please open one of the auto forwarded ports (80 and 443 should both work):
```
Comment (MarDipp) [inline .docs/codespaces.md:30]: I used this instructions to setup my first github codespace. It works fine, but at this point I thought I had to set up a port forwarding. But what is actually meant is to open one of the forwarded addresses in a browser window.

I think something like "please open one of the forwarded addresses in a new browser window" would be better

**Review by MarDipp — APPROVED** [review]
It´s cool ro run the escape room on github

File: .docs/codespaces.md (hunk 1 above)
Comment (peters-david) [reply .docs/codespaces.md:30]: Just to be clear about what to add and what to open, maybe something like:

Change to the ports tab and add one of the auto forwarded ports (80 or 443). A forwarded address should appear in the tab shortly after. Open the forwarded address in a new browser tab and add `/game/` to the url. The last slash is important.

File: .docs/codespaces.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+Add `/game/` to the url.
+(Right now the last slash is important).
+
+Broken parts in the study are marked with a red circle. 
+Hovering there with your cursor gives you a hint for what you might be looking for.
+
+![K8s Escape Room in web](../.images/k8s-escape-room-web-unsolved.png)
+
+After playing, the Github codespace should be [stopped](https://docs.github.com/en/codespaces/developing-in-a-codespace/stopping-and-starting-a-codespace?tool=webui#stopping-a-codespace) or deleted.
```
Comment (peters-david) [inline .docs/codespaces.md:42]: Optional: Maybe include direct link to codespaces for fast access?
https://github.com/codespaces

File: .docs/local.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+As soon as the cluster and its resources are ready you can start playing.
+
+To access the escape room UI in your browser please use [http://localhost/game/](http://localhost/game/).
+(Right now the last slash is important).
+
+Broken parts in the study are marked with a red circle. 
+Hovering there with your cursor gives you a hint for what you might be looking for.
+
+![K8s Escape Room in web](../.images/k8s-escape-room-web-unsolved.png)
+
```
Comment (peters-david) [inline .docs/local.md:47]: Are instructions how to have changes in code reflect in the cluster neccessary? Probably out of the scope of this pr?

**Review by peters-david — APPROVED** [review]
Everything worked during testing, thank you 😄

File: .docs/codespaces.md (hunk 1 above)
Comment (steadyk) [reply .docs/codespaces.md]: Thanks @MarDipp and @peters-david!
I've adapted the documentation based on your ideas! I hope this is now clearer! :-)

File: .docs/codespaces.md (hunk 2 above)
Comment (steadyk) [reply .docs/codespaces.md]: Thanks @peters-david! I've reworked the parts! :-)

**Comment by steadyk** [comment]
@MarDipp @peters-david  I've added a link to directly create a codespace.

**Comment by peters-david** [comment]
Looks great 👍🏼

**Pull request merged**